src/components/Landing/Landing.vue

```diff
@@ -352,6 +356,26 @@ export default {
         "https://api.etherscan.io/api?module=stats&action=tokensupply&contractaddress=0x898bad2774eb97cf6b94605677f43b41871410b1&apikey=GKKIY3WXXG1EICPRKACRR75MA4UE7ANFY8"
       );
       this.TVL = BN(response.data.result).div(1e18).toFixed(0).toString();
+
+      let token = SGT_uniswap;
```

Comment:
Thanks for doing this! 
For the sake of readability/clean code, can we please extract this into a standalone function? 
So we can use it like:   
```
this.APY = getAPY()
```

Reply:
created standalone function